mcr_scene_segmentation/ros/src/scene_segmentation_node.cpp

```diff
@@ -99,7 +112,52 @@ void SceneSegmentationNode::pointcloudCallback(const sensor_msgs::PointCloud2::P
         add_to_octree_ = false;
         event_out.data = "e_add_cloud_stopped";
         pub_event_out_.publish(event_out);
+
+    }
+
+    if (dataset_collection_)
+    {
```

Comment:
so now this will only segment if data collection is true?

Reply:
Yes, this param is used for dataset collection only. If we use normal pipeline, it will take a lot of time just for trigerring the node e.g. (e_start, e_add_cloud_start and e_segment), we wanted to reduce this for dataset collection using a singgle trigger. But I had a discussion with Santosh that we want to reduce the number of e_.., because there are too much already. Maybe we will remove this dataset collection trigger. I need a review from @sthoduka about this.

Reply:
Maybe an option is to create an action for `scene_segmentation`? Then you can specify enum-like constants like in `visualization_msgs/Marker` for the different tasks.

Reply:
I just requested the review for you

Reply:
For future reference, you can also do it directly: https://help.github.com/articles/requesting-a-pull-request-review/

It’s usually better that way, otherwise most people won’t get notified